package.json

```diff
@@ -14,8 +14,8 @@
     "node": ">=16.14.0"
   },
   "scripts": {
-    "start": "lerna run start --parallel --ignore @okta/odyssey-ie-11-sandbox",
-    "start:ie11": "lerna run start --scope @okta/odyssey-ie-11-sandbox",
+    "start": "yarn workspace @okta/odyssey-storybook start",
+    "start:ie11": "yarn workspace @okta/odyssey-ie-11-sandbox start",
```

Comment:
If you need parallel running, we also have a library already installed in some projects (like `okta-ui-react`) called `npm-run-all`. I use this myself for doing parallel operations. That way, you can run the same command across multiple workspaces, but it actually requires you create separate scripts for each command you want to run in parallel. Not exactly monorepo-friendly.